jsArrays/task2.js

```diff
@@ -0,0 +1,7 @@
+const initialArray = [1, 2, 3, 4, 5];
+
+const mutatedArray = initialArray.map((value, index) => {
```

Comment:
можна і так, але достатньо просто
```suggestion
const mutatedArray = initialArray.map((value, index) => value*index)```